Calculator_Project/Source.cpp

```diff
@@ -0,0 +1,251 @@
+#include <iostream>
+#include <vector>
+#include <iterator>
+#include <string>
+
+template <typename T>
+void vec_print(const std::vector<T>& container) {   //Print vector, entry by entry
+    for (const T equation : container)              //Found online, modified to generic type

```

Comment:
You should make this a const reference (`const T& equation`) rather than a const value to avoid copying objects just to print them.

If this is going to be used for just printing the equation string vector, I would make this just take a `std::vector<std::string>>` rather than making a template; otherwise, if it's going to be used more generically, I would change the name `equation` to `object` or something more generic to denote that the intention of the function is just to print the contents of a vector, whatever they may be.